src/Altinn.Notifications.Persistence/Repository/OrderRepository.cs

```diff
@@ -1,4 +1,6 @@
-﻿using Altinn.Notifications.Core.Models.NotificationTemplate;
```

Comment:
do we still need System.Text.Json here if using the static method?

Reply:
removed